### PR TITLE
GHA docs

### PR DESCRIPTION
**Pull request by loriab**
## Description
This autobuilds the docs and deploys them to a netlify site, https://psi4manual.netlify.app/index.html. It also starts correcting all the format and bad link warnings. Started with ~300 and down to 15, so ready to start reviewing. Once clean, we can turn on warnings->errors and avert future beaks. This hasn't started looking at docs content, only docs format errors. No linkchecker run yet either. GHA-wise, this is for every master commit. There'll be other GHA to run similarly on PRs so that ppl can preview docs changes or note when changes broke docs. But that'll need different permissions. This seems a good place to start. I don't think this needs minute review -- save your energies for real code.

## Todos

- [ ] Feature1
- [ ] Feature2

## Questions
- [ ] Any rst files added that I shouldn't have?

## Checklist
- [ ] Tests added for any new features
- [ ] [All or relevant fraction of full tests run](http://psicode.org/psi4manual/master/build_planning.html#how-to-run-a-subset-of-tests)

## Status
- [x] Ready for review
- [x] Ready for merge SQUASH

### CODE REVIEW

**Review by hokru — APPROVED** [review]

File: doc/sphinxman/source/chemps2.rst [hunk 1]
```diff
@@ -85,7 +85,7 @@ Installation
   and rebuild |PSIfour| to detect CheMPS2 and activate dependent code.
 
 * Previous bullet had details. To build |PSIfour| from source and use
```
Comment (JonathonMisiewicz) [inline doc/sphinxman/source/chemps2.rst:87]: Either remove the bullet point or add a new link.

File: doc/sphinxman/source/cppe.rst [hunk 2]
```diff
@@ -79,7 +79,7 @@ Installation
   and rebuild |PSIfour| to detect CPPE and activate dependent code.
 
 * Previous bullet had details. To build |PSIfour| from source and use
-  cppe from conda without thinking, consult :ref:`sec:condapsi4dev`.
```
Comment (JonathonMisiewicz) [inline doc/sphinxman/source/cppe.rst:82]: See above.

File: doc/sphinxman/source/gau2grid.rst [hunk 3]
```diff
@@ -74,7 +74,7 @@ Installation
   and rebuild |PSIfour| to detect gau2grid and activate dependent code.
 
 * Previous bullet had details. To build |PSIfour| from source and use
-  gau2grid from conda without thinking, consult :ref:`sec:condapsi4dev`.
```
Comment (JonathonMisiewicz) [inline doc/sphinxman/source/gau2grid.rst:77]: See above.

File: doc/sphinxman/source/gdma.rst [hunk 4]
```diff
@@ -73,7 +73,7 @@ Installation
   and rebuild |PSIfour| to detect gdma and activate dependent code.
 
 * Previous bullet had details. To build |PSIfour| from source and use
-  gdma from conda without thinking, consult :ref:`sec:condapsi4dev`.
```
Comment (JonathonMisiewicz) [inline doc/sphinxman/source/gdma.rst:76]: See above.

File: doc/sphinxman/source/libefp.rst [hunk 5]
```diff
@@ -76,7 +76,7 @@ Installation
   and rebuild |PSIfour| to detect libefp and activate dependent code.
 
 * Previous bullet had details. To build |PSIfour| from source and use
-  libefp from conda without thinking, consult :ref:`sec:condapsi4dev`.
```
Comment (JonathonMisiewicz) [inline doc/sphinxman/source/libefp.rst:79]: See above.

File: doc/sphinxman/source/libint.rst [hunk 6]
```diff
@@ -74,7 +74,7 @@ Installation
   and rebuild |PSIfour| to detect Libint and activate dependent code.
 
 * Previous bullet had details. To build |PSIfour| from source and use 
-  Libint from conda without thinking, consult :ref:`sec:condapsi4dev`.
```
Comment (JonathonMisiewicz) [inline doc/sphinxman/source/libint.rst:77]: See above.

File: doc/sphinxman/source/libxc.rst [hunk 7]
```diff
@@ -74,7 +74,7 @@ Installation
   and rebuild |PSIfour| to detect Libxc and activate dependent code.
 
 * Previous bullet had details. To build |PSIfour| from source and use
-  Libxc from conda without thinking, consult :ref:`sec:condapsi4dev`.
```
Comment (JonathonMisiewicz) [inline doc/sphinxman/source/libxc.rst:77]: See above.

File: doc/sphinxman/source/pcmsolver.rst [hunk 8]
```diff
@@ -76,7 +76,7 @@ Installation
   and rebuild |PSIfour| to detect PCMSolver and activate dependent code.
 
 * Previous bullet had details. To build |PSIfour| from source and use
-  pcmsolver from conda without thinking, consult :ref:`sec:condapsi4dev`.
```
Comment (JonathonMisiewicz) [inline doc/sphinxman/source/pcmsolver.rst:79]: See above.

File: doc/sphinxman/source/simint.rst [hunk 9]
```diff
@@ -81,13 +81,14 @@ Installation
   and rebuild |PSIfour| to detect simint and activate dependent code.
 
 * Previous bullet had details. To build |PSIfour| from source and use 
-  simint from conda without thinking, consult :ref:`sec:condapsi4dev`.
```
Comment (JonathonMisiewicz) [inline doc/sphinxman/source/simint.rst:84]: See above.

File: psi4/driver/procrouting/scf_proc/scf_iterator.py [hunk 10]
```diff
@@ -718,7 +716,7 @@ def scf_print_energies(self):
     #else:
     #    self.set_variable(self.functional().name() + ' TOTAL ENERGY', dft_energy)  # overwritten later for DH
 
-    self.set_variable('SCF ITERATIONS', self.iteration_)
+    self.set_variable("SCF ITERATIONS", self.iteration_)  # P::e SCF
```
Comment (JonathonMisiewicz) [inline psi4/driver/procrouting/scf_proc/scf_iterator.py:719]: There are a _lot_ of changes like this. Can you explain why this is necessary, and when we should add comments like this to a `set_variable` line?

**Review by JonathonMisiewicz — CHANGES_REQUESTED** [review]
A big step in the right direction, but there's a certain issue that occurs nine different times...

File: psi4/driver/procrouting/scf_proc/scf_iterator.py (hunk 10 above)
Comment (loriab) [reply psi4/driver/procrouting/scf_proc/scf_iterator.py:719]: we need to harvest qcvars from py-side and need to know what module they're assoc. with, so this comment is the marker for both. I've added some notes in the "docs 19" commit. good catch -- doesn't look like any of the scraping conventions were documented.

File: doc/sphinxman/source/chemps2.rst (hunk 1 above)
Comment (loriab) [reply doc/sphinxman/source/chemps2.rst]: yes, these were all referring to a nonexistant condapsi4dev docs section. I was planning on ditching all this repeat documentation, so the incomplete bullet wouldn't have lasted, but removing it is better for now.

**Review by JonathonMisiewicz — APPROVED** [review]

**Review by dgasmith — APPROVED** [review]

**Pull request merged**